app/assess/templates/components/application_overviews_table.html

```diff
@@ -0,0 +1,66 @@
+{% from "govuk_frontend_jinja/components/input/macro.html" import govukInput %}
+
+
+{% macro render(application_overviews) -%}
+
+<div data-module="fsd-filtered-fetch-table" data-endpoint="{{ application_search_endpoint }}"></div>
+<table class="govuk-table" data-module="sortable-table">
+
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+
+      <th scope="col" class="govuk-table__header">
+          Ref
+      </th>
+
+
+      <th scope="col" class="govuk-table__header">
+            Project name
+      </th>
+
+
+      <th scope="col" class="govuk-table__header">
+        Asset type
+      </th>
+
+
+      <th scope="col" class="govuk-table__header">
+        Total funding requested
+      </th>
+
+
+      <th scope="col" class="govuk-table__header">
+        Status
+      </th>
+
+    </tr>
+  </thead>
+
+  <tbody class="govuk-table__body">
+    {% if application_overviews %}
+        {% for overview in application_overviews %}
+            <tr class="govuk-table__row">
+
+                <td class="govuk-table__cell">{{overview.application_short_id}}</a></td>
+
+                <td class="govuk-table__cell"><a class="govuk-link" href="{{url_for('assess_bp.application',application_id=overview.application_id)}}">{{overview.project_name}}</a></td>
+                <td class="govuk-table__cell">{{overview.type_of_application}}</td>
+                <td class="govuk-table__cell">{{ "£%.2f"|format(overview.funding_amount_requested | int) }}</td>
+                {% if overview.workflow_status ==  "NOT STARTED" %}
```

Comment:
Think we need to add the styling for the status enums as here:  https://github.com/communitiesuk/funding-service-design-frontend/blob/main/app/templates/partials/tasklist_section.html#:~:text=class%3D%22govuk%2Dtag%20not%2Dstarted%2Dtag%22

Reply:
So we've put the styling in from what was in the prototype with that `span` element. Emma is reviewing the styles for the statuses for assessment anyway (just had a meeting) so will update once she's completed that.

Reply:
sounds good